swift/src/IceImpl/include/Connection.h

```diff
@@ -15,10 +15,7 @@ ICEIMPL_API @interface ICEConnection : ICELocalObject
 - (BOOL)setAdapter:(ICEObjectAdapter* _Nullable)oa error:(NSError* _Nullable* _Nullable)error;
 - (nullable ICEObjectAdapter*)getAdapter;
 - (ICEEndpoint*)getEndpoint;
-- (BOOL)flushBatchRequests:(uint8_t)compress error:(NSError* _Nullable* _Nullable)error;
-- (void)flushBatchRequestsAsync:(uint8_t)compress
-                      exception:(void (^)(NSError*))exception
-                           sent:(void (^_Nullable)(bool))sent;
+- (void)flushBatchRequests:(uint8_t)compress exception:(void (^)(NSError*))exception sent:(void (^_Nullable)(bool))sent;
```

Comment:
I don't thin sent should be nullable. We always supply it.

Reply:
I can make them non-nullable.